pkg/lease/manager_test.go

```diff
@@ -25,7 +25,7 @@ var NowTime = metav1.NowMicro()
 const (
 	leaseHolderIdentity = "some-operator"
 	leaseDuration       = 3600 * time.Second
-	leaseNamespace      = "some-lease-namespace"
+	leaseNamespace      = "medik8s-leases"
```

Comment:
I think this const is redundant @mshitrit

Reply:
I guess it's a matter of perspective.
You are correct that that we can use the const from the production code, however using a different const in test code decouples the test code from production code which also has some value.
IIRC @slintes usually prefer the later approach, I don't have a strong preference either way.